### PR TITLE
[KOGITO-4721] - Create Image distribution for Job Service MongoDB

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>
See: https://issues.redhat.com/browse/KOGITO-4721

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Review by vaibhavjainwiz — APPROVED** [review]

**Review by radtriste — CHANGES_REQUESTED** [review]
we need to keep the `kogito-jobs-service` in-memory as well (using `jobs-service-common` runner jar)

**Review by sutaakar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -623,28 +625,38 @@ To know what configurations this image accepts please take a look [here](kogito-
...
+docker run -it --env SCRIPT_DEBUG=true quay.io/kiegroup/kogito-jobs-service-infinispan:latest
 ```
+
 You should notice a few debug messages being printed in the system output.
 
-To know what configurations this image accepts please take a look [here](kogito-jobs-service-overrides.yaml) on the **envs** section.
+The ephemeral image does not have external dependencies like a backend persistence provider, it uses in-memory persistence
+while working with Jobs Services, while the `infinispan` and `mongodb` variants will need an Infinispan and MongoDB server,
+respectively, previously running.
+is not set to *true* it will use the default implementation (in-memory).
```
Comment (radtriste) [inline README.md]: leftover ?

**Review by radtriste — APPROVED** [review]
lgtm
one minor thing

File: modules/kogito-jobs-service-ephemeral/module.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+schema_version: 1
+name: org.kie.kogito.jobs.service.ephemeral
+version: "2.0.0-snapshot"
+
+artifacts:
+  - name: jobs-service-common-runner.jar
+    url: https://repository.jboss.org/nexus/content/groups/public/org/kie/kogito/jobs-service-common/2.0.0-SNAPSHOT/jobs-service-common-2.0.0-20210429.061304-7-runner.jar
```
Comment (radtriste) [inline modules/kogito-jobs-service-ephemeral/module.yaml]: `20210429` is pretty old. can't we have something newer ?

File: README.md (hunk 1 above)
Comment (spolti) [reply README.md]: yes, thanks for spotting it.

**Comment by spolti** [comment]
@ricardozanini have spotted an issue that is preventing the images to get built, PR is in place waiting for CI to pass:
https://github.com/kiegroup/kogito-images/pull/474

**Review by cristianonicolai — APPROVED** [review]

**Pull request merged**